lib/Mojolicious/Plugin/CGI.pm

```diff
@@ -215,10 +215,14 @@ sub register {
         $| = 1;
         select STDOUT;
         $| = 1;
-        { exec $self->{script} }
-        die "Could not execute $self->{script}: $!";
+	if ($self->{run}) {
+	  $self->{run}->();
```

Comment:
Huh? My idea is that you pass on $c to the code ref, to provide extra flexibility. I think passing on $self is a bad idea though. I guess ```$code->($c)``` is better.